book/website/community-handbook/governance.md

```diff
@@ -0,0 +1,61 @@
+(ch-governance)=
+# Governance and Decision Making in *The Turing Way*
+
+Governance is a set of formal and informal practices through which an organisation sets goals, assigns responsibilities, establishes systems, and assesses outcomes of organisational action.
```

Comment:
If governance can be both formal and informal, maybe we don't need to mention it.

Unless we want to emphasise that not all governance will be written down or clear to the community. In that case, it might be better in a separate sentence.

Maybe something like,

```suggestion
Governance is a set practices through which an community sets goals, assigns responsibilities, establishes systems, and assesses outcomes of organisational action.
Not all governance is formally described.
For example, processes may be understood by community leaders after evolving organically within a small group.
```